cvat/apps/engine/static/engine/js/shapeCreator.js

```diff
@@ -519,8 +524,8 @@ class ShapeCreatorView {
                     const { frameWidth } = window.cvat.player.geometry;
                     const { frameHeight } = window.cvat.player.geometry;
                     for (const point of actualPoints) {
-                        point.x = Math.clamp(point.x, 0, frameWidth);
-                        point.y = Math.clamp(point.y, 0, frameHeight);
+                        point.x = window.cvat.frameClipper.clamp(point.x, 0, frameWidth);
```

Comment:
nit: you can simplify this code now, no need to pass `frameWidth` and `frameHeight` as separate parameters - they are accessible from `frameClipper.clamp` itself.